### ISSUE TITLE
Suggested contributions

### ISSUE THREAD

**Issue by erelsgl** [fuglede/numberpartitioning#2]
Hi, are you still working on this project?

I have some code for k-way number partitioning, as well as for some related problems such as bin-packing and bin-covering. Will you be interested in adding them?

**Comment by fuglede** [comment]
Yep, it's active. And I think there's room for more functionality.

* The tiny bit of functionality that's already in here is also for multiway number partitioning (which happened to be what I needed myself). One thing that could be useful would be having more alternatives to KK and CKK (included in #1), perhaps taking inspiration from [this SO post](https://stackoverflow.com/q/32354215/5085211). Or perhaps something based on the MILP-solvers out there. I'm sitting on a few NumPy/Numba implementations for KK as well that give speed-ups in certain cases. I'd also consider PyO3/Cython if the packaging story didn't come with so much maintenance.
* For bin-packing and friends there are some established packages already -- e.g. [`binpacking`](https://github.com/benmaier/binpacking) -- and it would probably be less confusing/more efficient if further improvements went into that.
